iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -632,21 +589,31 @@ public static void checkTemporaryLimits(Validable validable, double permanentLim
             }
         }
         // check name unicity
+        AtomicReference<ValidationLevel> validationLevel = new AtomicReference<>(ValidationLevel.STEADY_STATE_HYPOTHESIS);
```

Comment:
Yes, that's fine with me.

Reply:
So we remove this check on names and we put a restriction to same tempo + value or only same tempo? @HugoKulesza

Reply:
As AbstractLoadingLimits are coded right now, we need to check the distinct tempo (to be used as keys in a map of temporary limits).
It would be useful in certain cases to change it to distinct (tempo, value) but it would require further refactoring.